docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,191 @@
+Verification
+============
+
+What is a verified tool/workflow?
+---------------------------------
+A verified tool/workflow indicates that it was successfully run and verified by either:
+
+-  the :ref:`Dockstore ToolTester`
+-  our team
+-  an outside party
+
+Currently, the majority of tool validation has been done by the
+`docktesters <https://lists.icgc.org/mailman/listinfo/docktesters>`__
```

Comment:
link probably won't work anymore, can just say something like 
`Historically, the majority of tool validation has been done by the docktesters team formerly headed by Miguel Vazquez and Francis Ouellette.`